mis_builder/views/mis_report_instance.xml

```diff
@@ -81,6 +81,7 @@
                                 <field name="date_range_id"/>
                                 <field name="date_from" attrs="{'required': [('comparison_mode', '=', False)]}"/>
                                 <field name="date_to" attrs="{'required': [('comparison_mode', '=', False)]}"/>
+                                <field name="analytic_account_id" groups="analytic.group_analytic_accounting"/>
```

Comment:
This field must move out of this group, because it's hidden in comparison mode.

Reply:
Well spotted !